genetic_algorithm/Main.cpp

```diff
@@ -18,10 +18,11 @@ int main()
 	Data::adjacencyMatrix =
 		std::vector <std::vector <int> >(n, std::vector <int>(n));
 
-	for (int i = 0; i < n; i++)
-		for (int j = 0; j < n; j++)
+	for (int i = 0; i < n; i++){
+		for (int j = 0; j < n; j++){
 			std::cin >> Data::adjacencyMatrix[i][j];
-
+		}
+	}
```

Comment:
Keep consistency with the bracket style (i.e.: use [Allman style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style)).